paddle/majel/test/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ cc_test(place_test
     SRCS place_test.cc
     DEPS majel)
 
+cc_test(ddim_test
```

Comment:
ok, to avoid clutter, I'm going to keep the folder structure unchanged temporarily and wait for the adjust in next PR.